core/modules/entity/entity.controller.inc

```diff
@@ -219,7 +219,7 @@ class DefaultEntityController implements EntityControllerInterface {
     // Reset the persistent cache.
     if ($this->persistentCache) {
       if (!empty($ids)) {
-        cache_clear_all($ids, 'cache_entity_' . $this->entityType);
+        cache('cache_entity_' . $this->entityType)->delete($ids);
```

Comment:
Although this works with the DatabaseCache backend, it's sort of a fluke of the database abstraction layer. Because `$ids` is an array, it should be:

```suggestion
        cache('cache_entity_' . $this->entityType)->deleteMultiple($ids);
```

I just ran into this line causing an error in the Backdrop port of memcache, which was expecting a single item and was getting passed an array.